src/game/obj_behaviors.c

```diff
@@ -642,7 +642,7 @@ void obj_check_floor_death(s16 collisionFlags, struct Surface *floor) {
             case SURFACE_BURNING:
                 o->oAction = OBJ_ACT_LAVA_DEATH;
                 break;
-            case SURFACE_VERTICAL_WIND: // fall through
+            case SURFACE_VERTICAL_WIND: FALL_THROUGH;
```

Comment:
A FALL_THROUGH isn't needed here, cases with no body do not warn when falling through.